pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -69,6 +69,8 @@ type SchedulingQueue interface {
 	Close()
 	// DeleteNominatedPodIfExists deletes nominatedPod from internal cache
 	DeleteNominatedPodIfExists(pod *v1.Pod)
+	// HasUnschedulablePods returns true if any unschedulable pods exist in the SchedulingQueue.
+	HasUnschedulablePods() bool
```

Comment:
Could you please change this to `NumUnschedulablePods() int` and return length of the queue? It might be useful for other functions in the future.

Reply:
Done